### PR TITLE
Update italian.json

### PR DESCRIPTION
**Pull request by LordRibblesdale**

### CODE REVIEW

File: i18n/italian.json [hunk 1]
```diff
@@ -423,218 +423,218 @@
...
+    "music-autoleave-now" : "Il mio tempo corrente prima di lasciare il canale per inattività è {}.",    
+    "music-autoleave-now-never" : "La mia impostazione auto-disconnessione per inattività è stata disattivata, non mi disconnetterò più per inattività.",    
+    "music-autoleave-unreasonable" : "Hai scelto un tempo tra {} e {} secondi se non sei un Patreon sul mio server.",
+    "music-autoleave-unreasonable-patron" : "Non puoi impostare un tempo superiore a {}, usa {} se vuoi che non mi disconnetta per inattività.",
+    "music-autoleave-never" : "Mai",    
+    "music-npmsg-reset" : "Il messaggio 'Sto riproducendo' all'inizio di ogni nuova canzone è stato inviato automaticamente in base a dove stai usando i comandi.",
+    "music-npmsg-set" : "Il messaggio 'Sto riproducendo' all'inizio della nuova canzone verrà inviata su {}.",
+    "music-npmsg-set-disabled" : "Il messaggio 'Sto riproduicendo' all'inizio della nuova canzone è disabilitato.",
+    "music-npmsg-not-found" : "Non trovo il canale di testo {}",
     "music-npmsg-now" : "Now playing message at the beginning of new song is now sent on {}.",
```
Comment (Zenrac) [inline i18n/italian.json]: Can you translate this line please ?

**Comment by LordRibblesdale** [comment]
;)

File: i18n/italian.json [hunk 2]
```diff
@@ -423,218 +423,218 @@
...
+    "music-plinfo-display" : "L'autoplaylist {} ha {} canzoni !",
+    "music-plinfo-warnings" : "Attenzione : Ci sono {} link inesistenti nella tua autoplaylist. Puoi rimuoverli tutti con {}",
+    "music-plinfo-warning" : "Attenzione : C'è un link inesistente nella tua autoplaylist. Puoi rimuoverlo facilmente con {}",
+    "music-plinfo-too-much" : "L'autoplaylist ha troppe canzoni per essere visualizzate qui. Ti invierò il contenuto dell'autoplaylist in DM in pochi secondi.",
+    "music-plrepair-someone" : "Puoi solo modificare le tue autoplaylist personali.",
+    "music-plrepair-loading" : "Sto riparando l'autoplaylist, attendere prego...",
+    "music-plrepair-all-ok" : "E' tutto ok, questa autoplaylist non necessita di essere riparata !",
+    "music-plrepair-removed-links" : "Completato, {} link inesistenti sono stati rimossi dall'autoplaylist.",
+    "music-plrepair-removed-link" : "Completato, `1` link inesistente è stato rimosso dall'autoplaylist.",
     "music-lyrics-title-not-find" : "Cannot find the title, please ensure your URL is correct.",
```
Comment (Zenrac) [inline i18n/italian.json]: Ohh and this one too please

Comment (LordRibblesdale) [reply i18n/italian.json]: Ok ;)

Comment (LordRibblesdale) [reply i18n/italian.json]: I edited directly as Propose change, sorry for creating duplicate PR

Comment (Zenrac) [reply i18n/italian.json]: Np

**Comment by Zenrac** [comment]
Thank you very much 👍

**Comment by LordRibblesdale** [comment]
Glad to help :P

**Pull request merged**